ios/Client/EXAppDelegate.m

```diff
@@ -53,7 +53,7 @@ - (void)_setUpUserInterfaceForApplication:(UIApplication *)application withLaunc
     return;
   }
   [[ExpoKit sharedInstance] registerRootViewControllerClass:[EXRootViewController class]];
-  [[ExpoKit sharedInstance] application:application didFinishLaunchingWithOptions:launchOptions];
+  [[ExpoKit sharedInstance] setUpApplication:application withLaunchOptions:launchOptions];
```

Comment:
Now that I think about it… shouldn't this method be called even if we are starting in headless mode (without `_window`)? We set `DDLog` infra and fatal handler there… 🤔

Reply:
Honesty, I don't know. Someone explicit checks if `window` exists. So, maybe we init handles mode differently?